examples/integral.jl

```diff
@@ -0,0 +1,88 @@
+using ITensorNumericalAnalysis
+
+using Graphs: SimpleGraph, uniform_tree
+using NamedGraphs: NamedGraph, rename_vertices
+using NamedGraphs.NamedGraphGenerators: named_grid, named_comb_tree
+using ITensors:
+  ITensors, ITensor, Index, siteinds, dim, tags, replaceprime!, MPO, MPS, inner, Op
+using ITensorNetworks: ITensorNetwork, dmrg, ttn, maxlinkdim, siteinds, union_all_inds
+using Dictionaries: Dictionary
+using Random: seed!
+using ITensors: Ops
+
+using UnicodePlots
+
+seed!(1234)
+L = 60
+#g = NamedGraph(SimpleGraph(uniform_tree(L)))
+#g = rename_vertices(v -> (v, 1), g)
+g = named_comb_tree((2, L ÷ 2))
+#g = named_grid((L,1))
+lastDigit = 1 - 1 / 2^(L ÷ 2)
```

Comment:
Clean up (remove `lastdigit` as I don't think it appears anywhere) and commented out lines